src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -2800,6 +2800,13 @@ else if (responseText.contains("Your crimbo tree is now 100% naked")) {
       RequestLogger.updateSessionLog("Your weighty thumb ring walloped.");
     }
 
+    // You breathe in the fresh mountain air and restore your soul.
+    if (responseText.contains("breathe in the fresh mountain air and restore your soul")) {
```

Comment:
Please add a test for this.

`canDetectPottedPlantWins` in `FightRequestTest.java` is a fine one to copy.